cli/src/cargo.rs

```diff
@@ -46,6 +41,8 @@ pub fn invoke(args: &Args) -> Result<()> {
 
     if args.no_capture {
         cargo_invocation.arg("--").arg("--nocapture");
+    } else {
+        cargo_invocation.arg("--quiet");
```

Comment:
This is to prevent cargo from logging progress messages ("Compiliing foo...", "Running unittests Bar" etc)